src/applications/disability-benefits/all-claims/content/form0781/workflowChoicePage.jsx

```diff
@@ -58,52 +62,51 @@ export const workflowChoicePageDescription = formData => {
   return (
     <>
       {conditionSelections(formData)}
-      <h4>When to consider adding this statement to your claim</h4>
       <p>
-        If you added a new mental health condition that’s related to your
-        military service, we encourage you to submit a statement. Mental health
-        conditions include conditions like PTSD or depression.
+        We encourage you to submit this optional statement for any new mental
+        health conditions related to a traumatic event that happened during your
+        military service. We’ll use the information to support your claim.
       </p>
       <p>
-        <strong>What to expect if you add a statement</strong>
-        <br />
-        We’ll ask you questions about:
+        <strong>Here’s what to know before you decide:</strong>
       </p>
       <ul>
         <li>
-          <strong>Traumatic events from your military service</strong>
+          You can submit this statement in 1 of 2 ways: Answer questions online
+          or upload a PDF version of the questions.
+        </li>
+        <li>
+          We’ll ask you to describe the traumatic events you experienced and how
+          they relate to your mental health conditions. You’ll likely need about
+          45 minutes to answer all the questions.
+        </li>
+        <li>
+          You can skip questions you can’t or don’t want to answer. And you can
+          save your in-progress online form anytime if you need a break.
         </li>
         <li>
-          <strong>Behavioral changes you experienced as a result</strong>
+          We’ll ask you to share supporting evidence (like health records) later
+          in the form.
         </li>
       </ul>
-
-      <p>
-        You will also be able to add{' '}
-        <strong>medical records, provider information,</strong> and{' '}
-        <strong> supporting documents </strong> later in the{' '}
-        <strong>Supporting Evidence </strong> section.
-      </p>
-      <p>
-        Answer as many or as few questions you feel comfortable answering. We’ll
-        use any information you share to support your claim.
-      </p>
-      <p>
-        To answer all the questions, you’ll likely need about 45 minutes. You
-        can answer the questions online. Or, you can fill out a PDF version of
-        the form and upload it as part of your online submission.
-      </p>
-      <p>
-        <va-link
-          external
-          href="https://www.va.gov/find-forms/about-form-21-0781/"
-          text="Get VA Form 21-0781 to download"
-        />
-      </p>
     </>
   );
 };
 
+export const workflowChoiceAdditionalDescription = (
+  <p>
+    <strong>Note:</strong> If you choose to fill out the PDF version, you can
+    download the PDF now.
+  </p>
+);
```

Comment:
This doesn't appear to be called anywhere. Perhaps you meant to utilize it further down in `sectionsOfMentalHealthStatement`, where the content is duplicated.